checker/dune

```diff
@@ -7,6 +7,7 @@
  (synopsis "Coq's Standalone Proof Checker")
  (modules :standard \ coqchk votour)
  (wrapped true)
+ (instrumentation (backend bisect_ppx))
```

Comment:
It feels wrong to have to put this stuff in all the dune files.

Reply:
How would you do it? A global switch was used before, but it had significant drawbacks.

Reply:
What drawbacks?

Reply:
Turns out people want to enable instrumentations per-library, not globally. There is more context in the discussion of the new syntax, I can look it up in the dune repos if you are interested.

Reply:
Note that this is a no-op in the regular case, it just adds the library for instrumentation when the instrumentation profile is working.